celery/fixups/django.py

```diff
@@ -77,6 +77,7 @@ def install(self) -> "DjangoFixup":
 
         self._settings = symbol_by_name('django.conf:settings')
         self.app.loader.now = self.now
+        self.app.task_cls = 'celery.contrib.django.task:Task'
```

Comment:
One problem at the moment (I think) is that it would override a custom class set by the user.

Reply:
do you have any alternative in mind?

Reply:
Not yet. I'm thinking that, prior to changing it, I could check the current value and don't change it if it's different than the default task. I don't think there is a use case for people overriding this specialised class back to the default one currently.

Reply:
I think I've fixed it now.